### ISSUE TITLE
BP-3179 External payments support

### ISSUE THREAD

**Issue by byr0n3** [buckaroo-it/BuckarooSDK_PHP#156]
There seems to be no way to create external payments using this SDK. I also noticed there doesn't seem to be any documentation on this, only on the Dutch docs. The documentation that is available however, is very barebones. [External payment docs](https://docs.buckaroo.io/docs/nl/externe-betaling)

It'd be nice if there was built-in support for this in the SDK and maybe the documentation on this could be expanded aswel.

**Comment by byr0n3** [comment]
After some tinkering I managed to locally create a custom payment provider for this that adds the required parameters as described in the docs. However, the API returns the following channel error:
`Channel 'WEB' not supported on action Pay for service externalpayment.`
This endpoint doesn't seem to be working as expected, yet it's still described in the documentation.

**Comment by byr0n3** [comment]
Thank you for your quick reply! I've managed to get this to work manually by creating a simple `ExternalPayment` class that extends from the `PayablePaymentMethod` class and adding the `backoffice` channel to the config of the `BuckarooClient`. I also noticed that the English docs have been updated with a section about External Payments and the note about the usage of a different channel, thanks for the quick update!

I think it might be nice to add a section about refunding an external payment in the docs. Simply changing the action to `Refund` and adding the `OriginalTransactionKey` does the trick but it might be overlooked.
